lazy_theta_star_b/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.5)
+project(lazy_theta_star_b)
+
+set(CMAKE_C_STANDARD 99)
```

Comment:
You should remove all these sets / compiler settings and instead pull in `nav2_common` which has a `nav2_package()` macro which will set what you need & setup coverage flags for lcov